Documentation/ExtensionArchitecture/HowTo/Events/Index.rst

```diff
@@ -0,0 +1,55 @@
+.. include:: /Includes.rst.txt
+.. index:: pair: Extension development; Events
+.. _extension-development-events:
+
+======
+Events
+======
+
+:ref:`PSR-14 events <EventDispatcher>` can be used to extend the TYPO3 Core
+or third-party extensions.
+
+You can find a complete list of events provided by the TYPO3 Core in the
+following chapter: :ref:`eventlist`.
+
+Events provided by third-party extensions should be described in the extensions
+manual. You can also search for events by searching for classes that inject the
+:ref:`EventDispatcherInterface`
+
+
+.. _extension-development-event-listener:
+
+Listen to an event
+==================
+
+If you want to use an event provided by the Core or a third party extension,
+create a PHP class with a method `public __invoke(SomeCoolEvent $event)`
+that accepts an object of the event class as
+argument. It is possible use another method name but you have to configure
+the name in the :file:`Configuration/Services.yaml` or it is not found.
+
+It is best practice to use a descriptive class name that ends on
+:file:`EventListener` and to put it in the namespace
+:php:`MyVendor\MyExtension\EventListener`.
+
+.. literalinclude:: _Joh316PasswordInvalidEventListener.php
```

Comment:
Maybe add a paragraph what is so special about "joh316" as password and why ist hosuld be prevented to use it (making some TYPO3 archaeology and history).

Reply:
added it as comment in the listener class